tests/lib/rule-tester/rule-tester.js

```diff
@@ -731,6 +731,41 @@ describe("RuleTester", () => {
 		});
 	});
 
+	describe("assert valid test cases", () => {
+		/**
+		 * Runs valid tests
+		 * @param {Array<string|object>} valid Valid tests
+		 * @returns {void}
+		 */
+		function runValidTests(valid) {
+			ruleTester.run(
+				"no-eval",
+				require("../../fixtures/testers/rule-tester/no-eval"),
+				{ valid, invalid: [] },
+			);
+		}
+
+		it("Valid test case must not have 'errors' property", () => {
+			assert.throws(() => {
+				runValidTests([{ code: "", errors: 1 }]);
+			}, /Valid test case must not have 'errors' property`/u);
+
+			assert.throws(() => {
+				runValidTests([{ code: "", errors: [{ message: "foo" }] }]);
+			}, /Valid test case must not have 'errors' property`/u);
+		});
```

Comment:
It would be good to also add tests with `errors: 0` (fails) and `errors: void 0` (passes).